tests/sentry/integrations/slack/test_notify_action.py

```diff
@@ -190,6 +190,43 @@ def test_invalid_channel_selected(self):
         assert not form.is_valid()
         assert len(form.errors) == 1
 
+    @responses.activate
+    def test_rate_limited_response(self):
```

Comment:
This tests that a 429 from slack when creating an issue alert is reflected in the form, so the user sees the error being surfaced.